src/test.js

```diff
@@ -1 +1,15 @@
-// empty file for right now
\ No newline at end of file
+// empty file for right now
+
+const Role = require('../src/role.js');
+const Player = require('../src/player.js');
+let role = new Role();
+let player = new Player();
+console.log(Role.colors);
+
+let players = ['1', '2', '3', '4', '5'];
+
+let assigned = player.assignPlayers(players, role.loadRoles())
```

Comment:
Something about this line makes me think that Game should assign players rather than the player class. 
Also the Game class could `loadRoles()`, and the list of roles would be roles from the Role class

Reply:
Yeah that makes better sense. Let's do it that way